HISTORY.md

```diff
@@ -1,3 +1,10 @@
+## 2.1.1 (March 28, 2024)
+
+### Bug Fixes
+
+- Fixes an issue with flattening the structs in the `utils.move_struct` method when `struct_name` contains a dictionary with `type_name` and `attributes`.
```

Comment:
This is a bit too inner-workings-of-the-SDK-esque... Let's go for:

> Fixes an issue where certain structs could cause errors during marshalling and unmarshalling

Reply:
Updated, thanks!